fixed bookshelf model/migration script bug

DBULL7 opened the pull request:
@alocke12992 I suspect this was a double whammy of errors. The folders didn't install correctly and then the script didn't exit when it detected something was wrong. Give it a shot by creating or adding a server with Postgres and try to make a model/migration.

Commit:
fixed bookshelf model/migration script bug

alocke12992 commented:
When creating a new backend project , and adding a model, the params passed in the console are being used to name the file and generate the model incorrectly. See the photo below; 

<img width="583" alt="screen shot 2018-09-28 at 8 31 00 am" src="https://user-images.githubusercontent.com/34324369/46218261-087f3600-c2f9-11e8-9831-6d55ff794e17.png">

alocke12992 commented:
Looks like I may have found a different bug, after digging into the code a bit I noticed that you're setting table columns to ```process.argv.splice(3, process.argv.length)```  but the data needed to create the columns would come from splitting argv[2]. Here is the console.log of argv: 

<img width="634" alt="screen shot 2018-09-28 at 8 41 04 am" src="https://user-images.githubusercontent.com/34324369/46219258-b5f34900-c2fb-11e8-8ee0-82e00f7a38d1.png">

Commit:
Merge branch 'v_2.0.0' of https://github.com/blixjs/blix into bookshelf-script-fix

Commit:
improved bookshelf model script, unable to reproduce bug andrew found

DBULL7 commented:
I've made the bookshelf model script a bit more legible. I still can't reproduce your bug, did you pull the latest or perhaps used comma's?

Commit:
Fixed process argv slice bug

Andrew discovered a bug in testing where the model command with args would create bad files when run with blix g. Fixed, the argument was previously a single string as an argv index, now it's split up into seperate argv arguments.

alocke12992 reviewed (approved):
Great work buddy. Bucko. Chief. Hats off. Rubba dub dub.
